client-v2/src/components/FrontsEdit/Front.tsx

```diff
@@ -236,23 +288,30 @@ class FrontComponent extends React.Component<FrontProps, FrontState> {
                 />
               </OverviewToggleContainer>
             </SectionContentMetaContainer>
-            <FrontCollectionsContainer>
+            <FrontCollectionsContainer
+              onScroll={this.handleScroll}
+              innerRef={ref => (this.collectionContainerRef = ref)}
```

Comment:
Quick thought - according to [React docs](https://reactjs.org/docs/refs-and-the-dom.html#dont-overuse-refs) there is a more modern way to do this... 
We're already doing it both ways in the code. Just preference but I'd argue the modern way with `React.createRef();` is more readable. 

Also! can we name the `ref` param as `element` so it doesn't get mixed up with `ref` the attribute?